lib/hanami/controller/configuration.rb

```diff
@@ -416,7 +416,7 @@ def format(hash)
       def mime_types
         @mime_types ||= begin
                           ((@formats.keys - DEFAULT_FORMATS.keys) +
-                          ::Rack::Mime::MIME_TYPES.values).freeze
+                          Hanami::Action::Mime::MIME_TYPES.values).freeze
```

Comment:
@mereghost If you agree, we can do `txt`, `html`, `json` in this order, and leave the rest as it is.